includes/REST/AdminDashboardController.php

```diff
@@ -164,6 +175,16 @@ public function get_feeds( $request ) {
         return rest_ensure_response( $feeds );
     }
 
+    public function get_status( $request ) {
+        /**
+         * @var Status $status
+         */
+        $status = dokan_get_container()->get( Status::class );
+        $content = $status->render();
+
+        return rest_ensure_response( $content );
+    }
+
```

Comment:
It would be good if we can define a standard for All API to apply filters. We can study the WP & WC.